main.py

```diff
@@ -42,7 +37,7 @@ def main():
     # Get all constraint property ids in a package as list
     cp_ids_list = xmlreader.find_constraint_property_ids()
```

Comment:
something_something_list is like hungarian notation. It is cleaner to just use the plural to indicate an iterable. So I am proposing: cp_ids_list -> cp_ids

Reply:
Whoops, overlooked it... Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/f9dc97172f7daab018167f85fdc90c1bf934e9f2